lib/web/sprite.js

```diff
@@ -70,7 +70,7 @@ function baseUrlWorkAround(svg, currentUrlPrefix, newUrlPrefix) {
       var attributeName = attribute.localName.toLowerCase();
 
       if (fixAttributes.indexOf(attributeName) !== -1) {
-        var match = URI_FUNC_REGEX.exec(node.getAttribute(attributeName));
+        var match = URI_FUNC_REGEX.exec(node.getAttribute(attributeName)).replace(/\\(.)/mg, "$1");
```

Comment:
I apologize, it isn't `getAttribute` that's escaping the string, it seems it's actually https://github.com/kisenka/svg-sprite-loader/blob/master/lib/web/sprite.js#L48 which is doing it. My thinking had been that this should allow deliberate backslashes such as `\\` just because I can't predict what each framework will try to use, and didn't want to push something that might fix the problem for Angular2 but create a problem for other frameworks. But now that I see the code is actually deliberately escaping parentheses already, I've pushed up the change to be

    var match = URI_FUNC_REGEX.exec(node.getAttribute(attributeName).replace(/\\\(/g, "(").replace(/\\\)/g, ")"));

so that escaped parentheses are handled. I can also just remove all backslashes like you suggest, although I think this solution is more consistent with what the `encodeUrlForEmbedding` function is doing.